home-manager.nix

```diff
@@ -40,7 +41,14 @@ let
 in
 {
   options = {
-
+    home.impermanence.enable = mkOption {
+      type = types.bool;
+      description = ''
+        Whether to enable impermanence for home-manager.
+        Defaults to "true".
+      '';
+      default = true;
```

Comment:
We could use `true` here, but add a warning to all new deployments and then change that to `false` after a reasonable amount of time? Same for the option below.